flutter/lib/benchmark/state.dart

```diff
@@ -359,7 +359,9 @@ class BenchmarkState extends ChangeNotifier {
         progressInfo.cooldown = true;
         final timer = Stopwatch()..start();
         progressInfo.calculateStageProgress = () {
-          return timer.elapsedMilliseconds / progressInfo.cooldownDurationMs;
+          return timer.elapsedMilliseconds /
```

Comment:
I didn't use it here because `inSeconds` getter returns int.
But, come to think of it, seconds precision should be enough for UI, so I guess it will be OK to change.